js/EventSystem.js

```diff
@@ -0,0 +1,43 @@
+class EventSystem {
```

Comment:
Ah just realized this file is missing the copyright

Reply:
Fixed